pkg/api/v1alpha1/cluster_types.go

```diff
@@ -566,6 +567,20 @@ const (
 	Kube124 KubernetesVersion = "1.24"
 )
 
+func GreaterOrEqual(kube1, kube2 KubernetesVersion) (bool, error) {
```

Comment:
Not sure if this should be in this package, it's an api package. This operation is not eks-a api specific, it's just comparing two minor semvers

Reply:
Continued in the below comment.